src/librustdoc/html/static/js/main.js

```diff
@@ -182,14 +202,120 @@ function hideThemeButtonState() {
     });
 }());
 
+/**
+ * This function inserts `newNode` after `referenceNode`. It doesn't work if `referenceNode`
+ * doesn't have a parent node.
+ *
+ * @param {DOM} newNode
```

Comment:
I think this is supposed to be HTMLElement (and same for other instances of "DOM" below).